ivadomed/transforms.py

```diff
@@ -118,6 +118,8 @@ def __init__(self, dict_transforms, requires_undo=False):
 
             # call transform
             if transform in globals():
+                if transform == "NumpyToTensor":
```

Comment:
note this fixes another bug but it needs to go in

Reply:
Could you perhaps add information on the bug it fixes in the PR description if it is not the same as #829. Thanks!

Reply:
done